install-cli.sh

```diff
@@ -77,11 +77,12 @@ findGoBinDirectory() {
             echo "Installation could not determine your \$GOPATH."
             exit 1
         fi
-        if [ -z "$GOBIN" ]; then
-            GOBIN=$(echo "${EFFECTIVE_GOPATH%%:*}/bin" | sed s#//*#/#g)
+        if [ -z "$GOROOT" ]; then
```

Comment:
why check on GOROOT is better than check on GOBIN?

Reply:
windows has no GOBIN?

Reply:
I prefer to completely get rid of GO env. We should install in three fixed dirs for macos, linux and windows.